packages/fx-core/templates/plugins/resource/bot/bicep/bot.template.bicep

```diff
@@ -122,5 +122,4 @@ output botDomain string = botWebAppHostname // validDomain
 output appServicePlanName string = botServerfarmsName // appServicePlan
 output botServiceName string = botServiceName // botChannelReg
 output botWebAppEndpoint string = botEndpoint // siteEndpoint
-output initiateLoginEndpoint string = initiateLoginEndpoint // redirectUri
```

Comment:
why deleted?

Reply:
This is the value for the redirect uri. Since we do not need this value, remove the output.